Merge two AutoOps FAQ pages into one (#3685)
elastic/docs-content: deploy-manage/monitor/autoops/ec-autoops-faq.md

* [Does AutoOps for self-managed clusters require additional payment?](#additional-payment)
* [Is there an added cost for shipping metrics data to {{ecloud}}?](#autoops-metrics-cost)
* [Which deployment types can be connected to AutoOps through Cloud Connect?](#deployment-types)

**Setting up AutoOps for self-managed clusters**
* [Can I use Cloud Connect to connect my {{ech}} clusters to AutoOps?](#cc-autoops-ech)
* [Can I use AutoOps for my clusters if my environment is air-gapped?](#autoops-air-gapped)
* [Can I use macOS to install {{agent}} for this feature?](#macos-install)
* [Do I have to define an Elastic IP address to enable the agent to send data to {{ecloud}}?](#elastic-ip-address)

**Collected metrics and data in AutoOps for self-managed clusters**
* [Where are metrics stored in AutoOps for self-managed clusters?](#autoops-metrics-storage)
* [What information does {{agent}} gather from my cluster?](#extracted-info)
* [How does AutoOps gather data from my cluster and ensure its security?](#data-gathering)
* [Can I view the data gathered by {{agent}}?](#data-viewing-config)

## General AutoOps questions

$$$what-is-autoops$$$**What does AutoOps do?**

> :   AutoOps for {{es}} significantly simplifies cluster management with performance recommendations, resource utilization, cost insights, real-time issue detection, and resolution paths. By analyzing hundreds of {{es}} metrics, your configuration, and usage patterns, AutoOps recommends operational and monitoring insights that deliver savings in administration time and hardware costs.

Reviewer: ```suggestion
:   AutoOps for {{es}} simplifies cluster management by providing performance recommendations, resource utilization and cost insights, real-time issue detection, and resolution paths. By analyzing hundreds of {{es}} metrics, your configuration, and usage patterns, AutoOps provides operational and monitoring recommendations that reduce administration time and hardware costs.
```

    :::{include} /deploy-manage/_snippets/autoops-cc-payment-faq.md
    ::: 

$$$autoops-monitoring$$$**Does AutoOps monitor the entire {{stack}}?**
:   AutoOps is currently limited to {{es}} (not {{kib}}, Logstash, or Beats).

$$$autoops-issue-resolution$$$**Can AutoOps automatically resolve issues?**
:   AutoOps only analyzes metrics and is a read-only solution.

$$$autoops-supported-versions$$$**Which versions of {{es}} are supported in AutoOps for {{ech}} and self-managed clusters?**
:   AutoOps is compatible with [supported {{es}} versions](https://www.elastic.co/support/eol) (7.17.x and above).

$$$autoops-data-retention$$$**How long does Elastic retain AutoOps data?**
:   Currently, AutoOps has a 10 day retention period.

$$$autoops-metrics-storage$$$**Where are AutoOps metrics stored, and does AutoOps affect customer ECU usage?**
:   AutoOps metrics are stored internally within the Elastic infrastructure, not on customer deployments. Using AutoOps does not consume customer ECU.

$$$autoops-vs-stack-monitoring$$$**Has AutoOps replaced Stack Monitoring?**

> :   Currently, AutoOps has many of the same features as Stack Monitoring as well as several new ones. However, it only provides insights on {{es}} and analyzes metrics, but not logs. Read more in [](/deploy-manage/monitor/autoops-vs-stack-monitoring.md).

Reviewer: ```suggestion
:   Currently, AutoOps has many of the same features as Stack Monitoring as well as several new ones. However, it only provides insights on {{es}} and analyzes metrics, not logs. Read more in [](/deploy-manage/monitor/autoops-vs-stack-monitoring.md).
```
